frontend/src/main/components/Quarters/SingleQuarterDropdown.js

```diff
@@ -20,6 +20,9 @@ function SingleQuarterDropdown({
   onChange = null,
   label = "Quarter",
 }) {
+  if (localStorage.getItem("PersonalScheduleForm-quarter") == null) {
+    localStorage.setItem("PersonalScheduleForm-quarter", quarters[0].yyyyq);
```

Comment:
Components cannot have side effects during render. For more discussion on this see https://github.com/ucsb-cs156-f23/proj-courses-f23-7pm-4/pull/26#discussion_r1401336976